visual-challenge/src/main/java/applitools/hackathon/sections/LoginPage.java

```diff
@@ -0,0 +1,58 @@
+package applitools.hackathon.sections;
+
+/**
+ * Created by Roman Iovlev on 10.11.2018
+ * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
+ */
+
+import com.epam.jdi.light.asserts.core.SoftAssert;
+import com.epam.jdi.light.elements.common.Label;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.ByText;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.*;
+
+public class LoginPage extends WebPage {
+    @UI(".auth-header") Label header;
+    @UI(".logo-w img") Image circleImage;
+    @UI(".alert-warning") public Text alert;
+    @ByText("Username") Label userLabel;
+    @ByText("Password") Label passwordLabel;
+    @UI(".os-icon-user-male-circle") Icon userIcon;
+    @UI(".os-icon-fingerprint") Icon fingerprintIcon;
+    @UI(".form-check-label") Checkbox rememberMe;
+    @UI("[src*=twitter]") Icon twitter;
+    @UI("[src*=facebook]") Icon facebook;
+    @UI("[src*=linkedin]") Icon linkedin;
+
+    TextField username, password;
+    Button logIn;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4513258)